app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({
+  setContacts,
+  onClose,
+  selectedContactId,
+  setSelectedContactId,
+}) => {
+  const [deleting, setDeleting] = useState(false);
+
+  const handleDelete = () => {
+    setDeleting(true);
+    setContacts(prevContacts =>
+      prevContacts.filter(contact => contact.id !== selectedContactId)
+    );
+    Toastr.success("Contact deleted succesfully");
+    setSelectedContactId();
```

Comment:
Not sure why am empty setter is being called. This is an anti-pattern. All setters are supposed to be passed in some value.